tests/annotations_test.go

```diff
@@ -0,0 +1,44 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// ensures services.annotations values are applied to both coderd deployment &
+// service
+func TestAnnotations(t *testing.T) {
+	t.Parallel()
+
+	chart := LoadChart(t)
+
+	expected := map[string]string{}
+
+	objs := chart.MustRender(t, func(cv *CoderValues) {
+		cv.Coderd.Annotations = expected
+		cv.Coderd.ServiceSpec.Annotations = expected
+		cv.Postgres.Default.Annotations = expected
```

Comment:
@deansheather this accounts for each annotation value being set properly in the Helm chart. I'd be curious to know if there's a better way to set multiple `struct` values to `expected`.

Reply:
Not sure what your question is, but this looks OK to me

Reply:
cool - I just wasn't sure if this was the most efficient method in setting multiple values to the `expected` variable.